src/overview.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: Overview
+---
+
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+store your website's resources locally, on your users' devices. Consider Workbox
+if you want to:
+
+* Make your site work offline.
+* Improve load performance on repeat-visits. You can use Workbox to store
+  and serve common resources locally, rather than from the network.
+
+The only difference between the two options is whether you store all or only
```

Comment:
I'd remove this line about the difference. Again, if we're pointing out things that you could do with Workbox that you couldn't do traditionally, the distinction is that you can now respond to navigation requests without having to go to the network.